api/goldens/aptos_api::tests::events_test::test_get_events.json

```diff
@@ -1,10 +1 @@
-[
-  {
-    "key": "0x04000000000000000000000000000000000000000a550c18",
-    "sequence_number": "0",
-    "type": "0x1::Reconfiguration::NewEpochEvent",
-    "data": {
-      "epoch": "1"
-    }
-  }
-]
+[]
```

Comment:
this seems wrong

Reply:
updated the address and regenerate the golden file